packages/itwin/map-layers/src/PreferencesBrowserStorage.ts

```diff
@@ -0,0 +1,125 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { ITwinIdArg, PreferenceArg, PreferenceKeyArg, UserPreferencesAccess } from "@itwin/core-frontend";
+
+/** Index signature holding preferences content
+ *
+ * @internal
+ */
+interface KeyContentProps {
+  [preferenceKey: string]: any;
+}
+
+/**
+* @beta
+*/
+export interface PreferencesBrowserStorageOptions {
+  /** indicate whether or not delete function should throw an error if the provided key cannot be found. */
+  throwOnDeleteMissingKey?: boolean;
+}
+
+/** User preferences implementation using browser's local storage.
+  * It creates storage item key in the form of:
+  *     <storageItemKeyPrefix>.<project_guid?>.<imodel_guid?>
```

Comment:
we're not using the term project anymore, can you replace with itwin?

Reply:
L27 too

Reply:
@aruniverse should be good now...